src/components/common/SelectPhotos.tsx

```diff
@@ -13,10 +13,10 @@ export default function SelectPhotos(props: PhotoChooserProps): JSX.Element {
   return (
     <PhotoChooser
       {...props}
-      uploadText={strings.UPLOAD_PHOTOS}
```

Comment:
We are using this selector for withdrawals and reports. Are we sure we want to change the upload text in all the views?